.circleci/config.yml

```diff
@@ -100,16 +99,17 @@ jobs:
       - run:
           name: Make binaries executable
           command: |
-            chmod +x out/*
+            chmod +x node/dist/*
       - run:
           name: Package
           command: >
-            RELEASE_TAG=v$(cat package.json | jq --raw-output
-            .version)_$CIRCLE_BUILD_NUM
+            RELEASE_TAG=node.v14.16.1_$GITHUB_SHA
```

Comment:
future, we might consider making the `14.16.1` a constant somewhere, possibly for other version?

Reply:
Yah. I plan on turning this into a Makefile, or a script, or something. My first goal is to build tis thing once and forget about it :D